README.md

```diff
@@ -91,7 +91,7 @@ stack --docker --docker-image=$IMAGE_NAME build --exec Main
 If you want to use GPU you can do:
 
 ```
-IMAGE_NAME=tensorflow/haskell:1.9.0-gpu
+IMAGE_NAME=tensorflow/haskell:1.4.0-gpu
```

Comment:
This looks like a downgrade. Undo?

Reply:
Yes, that's a typo. Thanks for noticing!